### ISSUE TITLE
Define the Advisory Board

### ISSUE THREAD

**Issue by ktjnyc** [hackforla/food-oasis#937]
### Overview
We need to create an advisory board so that we have experts able to guide and validate our product/updates.  

### Action Items

- [ ] Define a very clear/specific role or what is our ask of each person (maybe UX team can help define what we need)
- [ ] Define what the commitment ask is (number of meetings/hours - 4 lunches/year or 10 hours of advice, periodic survey, etc)
- [ ] Potentially run this list by Pearson for feedback 
- [ ] User advisory board may be separate here 
- [ ] Create a role to manage community outreach group - maintaining the relationships above
- [ ] Move #873 to prioritized backlog once this is complete

### Resources/Instructions
[List of potential people/organizations](https://docs.google.com/document/d/1FGMc76ta969LTGHQQn-aIXEi1A8Uyosk_y5EmbBL-qI/edit?usp=sharing)
[Draft advisory board guide](https://docs.google.com/document/d/1fC2jH3qRFWXkonVk5st49CAdxVt8c5xZL3RwICFx-rI/edit?usp=sharing)

**Comment by ktjnyc** [comment]
Surveys

**Comment by wanderingspoon** [comment]
I recommend being as specific as possible about areas of expertise required and expected contributions. Also very helpful would be language outlining the benefits of their involvement for themselves as advisors and ultimately for food seekers in LA. 

Some inspiration for describing advisory board positions...

[Sample: Advisory Council Job Description](https://www.ceffect.com/wp-content/uploads/2014/03/advisory.pdf)

[Advisory Board: descriptions of various positions](https://www.cteworksminnesota.org/wp-content/uploads/2016/09/mary-toner-workshop-handouts.pdf)

[Good example of simple structure with informal but useful headings](https://www.allamadvisorygroup.com/careers/aag-board-advisor-position/)

@ktjnyc

**Comment by ktjnyc** [comment]
Progress: None, can review Thy's comments by next Thursday
Blockers: Availability
Availability: A few hours tomorrow and possibly next week 
ETA: can review Thy's comments by next Thursday and make additional updates

**Comment by staceyrebekahscott** [comment]
Store names / contact info of any person who is a potential resource for FOLA

- [ ]  Need to locate shared drive document for ongoing contact list to update
- [ ]  How do we keep track of our engagement / information received with such contacts?

The checkboxes above were identified in issue #1250 , which has been closed because it overlaps with the objective of this issue.

**Comment by staceyrebekahscott** [comment]
At this stage of the project, we should be seeking advice from partners and users rather than a board of advisors. A new plan for partnership engagement is in process for the Impact Sprints 2022. 

There may be a need to develop an advisory board in the future, but I don't think it needs to be an issue at this time. 

Closing this issue as outdated.
